PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkLocalRunCommonParametersPanel.java

```diff
@@ -52,7 +53,9 @@ public String getWorkingDirectory() {
             myEnvVariablesComponentField.setAccessible(true);
             return ((EnvironmentVariablesComponent) myEnvVariablesComponentField.get(this)).getEnvs();
         } catch (NoSuchFieldException | IllegalAccessException e) {
-            throw new RuntimeException(e);
+            return new HashMap<String, String>() {{
+                put("HADOOP_HOME", System.getenv("HADOOP_HOME"));
+            }};
```

Comment:
This looks hack. Can you move the logic to "try" block, like "if containsKey() == false, System.getenv"

Reply:
I'll revert the code change to rethink of it.